src/index.html

```diff
@@ -1,13 +1,37 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
+
+  </head>
+  <body>
+    <container class="product-card" data-qa="card">
+      <img class="product-image"
+       src="images/imac.jpeg"
+       alt="Product Image">
+      <h2 class="product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <p class="product-code">Product code: <span>195434</span></p>
+      <div class="product-rating-and-reviews">
+        <div class="product-rating">
+          <div class="star"></div>
```

Comment:
```suggestion
        <div class="stars stars--4>
          <div class="stars__star"></div>
```

Fix due to BEM and [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)